.github/workflows/chez-build.yml

```diff
@@ -36,15 +36,14 @@ jobs:
           fetch-depth: 50
       - name: Download pb boot files
         run: make fetch-pb
-      - name: Proceed with TARGET_MACHINE == ${{ matrix.mach }}
-        run: echo $TARGET_MACHINE
       - name: Install libs for 32-bit
         if: matrix.mach == 'i3le' || matrix.mach == 'ti3le'
         run: |
           sudo dpkg --add-architecture i386
           sudo apt-get update
           sudo apt-get install -y gcc-multilib lib32ncurses-dev libssl-dev:i386
       - name: Install libs for 64-bit
+        # if: ???
```

Comment:
@pmatos

Should there be an `if: ...` here to restrict the step to only 64-bit environments?

Reply:
Yes, I think so  but how come this is not failing... I guess, the libraries names probably exist anyway in 32 bit envs.

Reply:
Thanks. I will add an `if: ...` clause then, so it seems appropriate to do so.